### PR TITLE
Solves issue #3 and new isValidRequest method

### PR DESCRIPTION
**Pull request by emmgfx**

### CODE REVIEW

File: CSRF_Protect.php [hunk 1]
```diff
@@ -68,11 +68,25 @@ public function echoInputField()
...
+
+	/**
+	 * Verifies whether the post token was set, and return true or false
+	 */
+	public function isValidRequest()
+	{
+		if(!isset($_POST[$this->namespace]))
+			return false;
+		
+		if (!$this->isTokenValid($_POST[$this->namespace]))
```
Comment (XCMer) [inline CSRF_Protect.php]: I think this line can just be `return $this->isTokenValid($_POST[$this->namespace]`

**Comment by emmgfx** [comment]
The problem is that the `$this->namespace` index on the `$_POST[]` array can be empty, and using it directly can generate a Notice with the error (and break a json response, Eg.)

And the `isValidRequest` is used to give a boolean response that don't terminate the process. The two ways are valid for some cases.

**Comment by emmgfx** [comment]
By the way, great class.

**Comment by XCMer** [comment]
Yes, I understood. I was talking about the call inside of your isValidRequest function.

**Comment by emmgfx** [comment]
Oh, yes. You're OK. This line can be changed if you want. :+1:

**Comment by emmgfx** [comment]
I added a new commit to this pull request : ) I thing that this should be merged.

Greetings.

**Pull request closed without merging**